.github/ISSUE_TEMPLATE/doc-issue.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Documentation/tutorial notebooks
+about: Let us know what would make the GPflow documentation better
+---
+
+# Documentation/tutorial notebooks
+
+*Is there anything missing in the docs?*
+
+*Are there any mistakes in the docs?*
+
+*Is there a feature that needs some example code in a notebook?*
+
+*Do you know how to fix the docs?* If so, it'd be amazing if you'd be willing to directly contribute a pull request :)
```

Comment:
you've used html comments above but bullets here. how come?

Reply:
Because there's less structure (and less structure needed) in the doc issues template, and it seemed a bit overkill / too heavy for this one.